pkg/agent/test/mockapi/storageinterface.go

```diff
@@ -0,0 +1,128 @@
+// storageinterface includes gcs related interfaces, which can be used to wrap
+// a type which implements the given methods. This is useful for testing, where mocking
+// needs to be done for the gcs client.
+
+// This class was referenced from googleapis/google-cloud-go-testing
+
+package mockapi
+
+import (
+	"context"
+	"io"
+	"cloud.google.com/go/iam"
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+)
+
+type Client interface {
```

Comment:
@mszacillo Is there any reason to copy over the interface ? Seems like we can use https://github.com/googleapis/google-cloud-go-testing/blob/master/storage/stiface/interfaces.go

Reply:
@yuzisun That was the original plan, but there were conflicts upon importing the google-cloud-go-testing library. So I temporarily included these as a workaround in order to create some unit tests. However, after updating and rebasing on master I noticed that the import works now - so I'll go ahead and remove these!